### PR TITLE
Feat/fill review form with be data

### PR DESCRIPTION
**Pull request by YF91925352**
# Description
Fill the reviews form with BE endpoint:reviews/:id/uid/:uid data.But here I use a fixed uid data temporary.After we can add reviews,I will change it to a variable user.uid
# Jira task link
https://hackyourfuture-dk.atlassian.net/jira/software/projects/VIEW/boards/7?selectedIssue=VIEW-66
# Screenshots (if applicable)
Screenshot 2023-05-23 at 15 15 14

### CODE REVIEW

File: packages/client/src/components/BigMovieCard/BigMovieCard.jsx [hunk 1]
```diff
@@ -48,6 +57,12 @@ export const BigMovieCard = ({ currentMovie }) => {
     borderRadius: '1.25rem',
     margin: '2.5rem 0',
   });
+  // eslint-disable-next-line no-console
```
Comment (mohamedaldahoul) [inline packages/client/src/components/BigMovieCard/BigMovieCard.jsx:60]: Can you remove the comment and the console.log, pls?

Comment (YF91925352) [reply packages/client/src/components/BigMovieCard/BigMovieCard.jsx]: Hi,I have deleted them~

File: packages/client/src/components/RatingStars/RatingStars.jsx [hunk 2]
```diff
@@ -6,40 +6,32 @@ import styled from '@emotion/styled';
...
   ratingText,
   handleOpenReview,
+  alignSelf,
 }) => {
+  const StarsRatingWrapper = styled(Box)`
+    align-self: ${alignSelf ?? ''};
+  `;
+
   return (
     <StarsRatingWrapper
```
[the remark this answers is not in the document]
Comment (YF91925352) [reply packages/client/src/components/RatingStars/RatingStars.jsx:19]: Sorry,do you mean here we don't need to create the variable StarsRatingWrapper.Instaed,I can write the styles in the Box:like this directly?
<Box
      sx={{
         align-self: ${alignSelf ?? ''} 
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'flex-end',
      }}
    >

**Review by mohamedaldahoul — APPROVED** [review]
Looks good

**Pull request merged**